### DISCUSSION TITLE
Final project query

### DISCUSSION THREAD

**Discussion by leslyarun** [OpenMined/courses#201] [Project: Privacy Product Specification] [answered]
For the final project, should we write about an existing product/service which leverages privacy AI or it should be a new idea?

**Comment by yemikifouly** [comment, marked as answer]
Hello Lesly! 👋 For the project it should be an idea of yours as if you were making a proposal for a product or service. 🙂 It doesn't have to be something completely new. Instead, it could be something that already exists but would be much better and safer with privacy principles applied to it. So, in that case, you could take the original idea and re-work it so that it applies privacy principles and write it up as your proposal. Additionally, here are a few helpful resources:

* **[Trask's idea suggestion](https://openmined.slack.com/archives/C01HY1GAG76/p1609700460117000)**.
* **[Kyoko's tip on coming up with a great idea](https://openmined.slack.com/archives/C01HY1GAG76/p1609867042192000?thread_ts=1609849270.187600&cid=C01HY1GAG76)**.

Note that both link redirect you to our Slack workspace. If you haven't joined yet, you will need to join. 🙂 I hope these help.
